core/http/src/main/java/org/trellisldp/http/impl/OptionsHandler.java

```diff
@@ -124,8 +127,8 @@ public ResponseBuilder ldpOptions(final ResponseBuilder builder) {
             } else {
                 // Containers support POST
                 builder.header(ALLOW, join(",", GET, HEAD, OPTIONS, PATCH, PUT, DELETE, POST));
-                builder.header(ACCEPT_POST, getServices().getIOService().supportedWriteSyntaxes().stream()
-                        .map(RDFSyntax::mediaType).collect(joining(",")));
+                builder.header(ACCEPT_POST, concat(getServices().getIOService().supportedWriteSyntaxes().stream()
```

Comment:
IIUC, you're concating `WILDCARD` onto a stream from `getServices().getIOService().supportedWriteSyntaxes()`, but this is getting a bit hard to read. How about:
```
rdfSyntaxes = getServices().getIOService().supportedWriteSyntaxes();
allSyntaxes = concat(rdfSyntaxes.stream().map(RDFSyntax::mediaType), of(WILDCARD));
builder.header(ACCEPT_POST, allSyntaxes.collect(joining(",")));
```
modulo appropriate type declarations, or something a little more broken-out?

Reply:
Yes, that would make it more readable.